ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -446,6 +458,37 @@ internal sealed class ArchiWebHandler : IDisposable {
 			return await UrlGetToHtmlDocumentWithSession(SteamCommunityURL, request).ConfigureAwait(false);
 		}
 
+		internal async Task<HashSet<ulong>> GetDigitalGiftCards() {
+			const string requestGifts = "/gifts";
+			HtmlDocument response = await UrlGetToHtmlDocumentWithSession(SteamStoreURL, requestGifts).ConfigureAwait(false);
+			if (response == null) {
+				return null;
+			}
+
+			HtmlNodeCollection nodes = response.DocumentNode.SelectNodes("//div[@class='pending_gift']/div[starts-with(@id,'pending_gift_')][count(div[@class='pending_giftcard_leftcol'])>0]/@id");
+			HashSet<ulong> results = new HashSet<ulong>();
+			foreach (string gidText in nodes.Select(node => node.GetAttributeValue("id", null))) {
+				if (string.IsNullOrEmpty(gidText)) {
+					Bot.ArchiLogger.LogNullError(nameof(gidText));
+					return null;
+				}
+
+				if (gidText.Length <= 13) {
+					Bot.ArchiLogger.LogGenericWarning(string.Format(Strings.ErrorIsInvalid, nameof(gidText)));
```

Comment:
This should also be generic error as we do not expect gid to have length less than 13.